src/experiments/data/actions.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
+import { EXPERIMENT_NAME_PREFIX } from './constants';
+
+export function toggleExperiment(experimentName) {
+	const storageItem = JSON.parse(
+		window.localStorage.getItem(EXPERIMENT_NAME_PREFIX + experimentName)
+	);
+
+	const newVariation =
+		storageItem.variationName === 'control' ? 'treatment' : 'control';
+
+	storageItem.variationName = newVariation;
+
+	window.localStorage.setItem(
```

Comment:
I think we can use the [Persistence Plugin](https://github.com/WordPress/gutenberg/tree/trunk/packages/data/src/plugins/persistence
), If we have time to look into it. Because it's usually not a best practice to put side-effect logic in the actions.

- [Where to write to localStorage in a Redux app?](https://stackoverflow.com/questions/35305661/where-to-write-to-localstorage-in-a-redux-app)

Reply:
Thank you for the suggestion. I wasn't aware of the Persistence Plugin 👍 

So we're not actually saving/loading states from localStorage. The data from localStorage is actually from ExPlat. Persistence Plugin doesn't fit here since not trying to save our states into external storage.

The code in this PR gets the experiments list from localStorage for display purposes.

I think this is the simplest approach for this problem, but I'm open to any suggestions.